tests/51media/01unicode.pl

```diff
@@ -44,11 +75,15 @@
 
 sub test_using_client
 {
-   my ( $client ) = @_;
+   my ( $client, $content ) = @_;
+
+   if( ! defined( $content )) {
```

Comment:
I'd be tempted to write

    unless( defined $content )

but this is fine too